model-mesh/rbac/kustomization.yaml

```diff
@@ -0,0 +1,36 @@
+# Copyright 2021 IBM Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+resources:
+  - service-account.yaml
+  - role.yaml
+  - role_binding.yaml
+  - leader_election_role.yaml
+  - leader_election_role_binding.yaml
+  - restricted_scc_role.yaml
+  - restricted_scc_role_binding.yaml
+  # - predictor_editor_role.yaml
+  # - predictor_viewer_role.yaml
+  # - servingruntime_editor_role.yaml
+  # - servingruntime_viewer_role.yaml
+  - modelmesh-service-account.yaml
+  - networkpolicy-controller.yaml
+  - networkpolicy-runtimes.yaml
+# Comment the following 4 lines if you want to disable
+# the auth proxy (https://github.com/brancz/kube-rbac-proxy)
+# which protects your /metrics endpoint.
```

Comment:
we should uncomment the following lines -- having metrics behind proxy makes sense to me.